uwblueprint/rowan-house
# Loading real courses

## Implementation Description
Load in real courses as opposed to dummy data on the courses view page.

## Screenshots
Visual depiction of PR
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/36215359/188338775-b68d60a3-23ee-4189-8b2c-f7939e55ddca.png">


## What should reviewers focus on?
- Make sure that real courses are now being displayed at the root URL (localhost:3000)

## Testing Procedure
- Create some courses, note the title and the description
- Observe that when you go to the root URL this is what is being displayed

## Checklist
